src/test/java/com/revature/services/impl/CarServiceImplTest.java

```diff
@@ -1,21 +1,22 @@
 package com.revature.services.impl;
 
-import com.revature.models.Car;
-import com.revature.models.User;
-import com.revature.repositories.CarRepository;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+import static org.mockito.Mockito.when;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.mockito.InjectMocks;
 import org.mockito.Mock;
 import org.springframework.test.context.junit4.SpringRunner;
 
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Optional;
-
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.fail;
-import static org.mockito.Mockito.when;
+import com.revature.models.Car;
+import com.revature.models.User;
```

Comment:
The changes in this file seem irrelevant. Please remove.

Reply:
Removed irrelevant changes.